examples/multipole/README.rst

```diff
@@ -16,3 +16,9 @@ In this test, the initial and final values of :math:`\sigma_x`, :math:`\sigma_y`
 .. literalinclude:: input_multipole.in
    :language: ini
    :caption: You can copy this file from ``examples/multipole/input_multipole.in``.
+
+This test can also be run as a Python script (``python3 run_multipole.py``):
```

Comment:
Please put Python above the inputs - see FODO example :)

Reply:
we decided to do this in a separate issue / PR